app/views/payment_gateways/_braintree_customer_form.html.erb

```diff
@@ -129,6 +129,7 @@
 
 <script src="https://js.braintreegateway.com/web/3.69.0/js/client.min.js"></script>
 <script src="https://js.braintreegateway.com/web/3.69.0/js/hosted-fields.min.js"></script>
+<script src="https://js.braintreegateway.com/web/3.69.0/js/three-d-secure.js"></script>
```

Comment:
Lets use the latest version 3.71 maybe? or put it in another PR?

Reply:
Better to extract this kind of risky changes to its own PR. In this case, this version is already in https://github.com/3scale/porta/pull/2310

Reply:
upgraded to the latest version in [#2322](https://github.com/3scale/porta/pull/2322)